java/ql/lib/semmle/code/xml/Ant.qll

```diff
@@ -9,7 +9,7 @@ class AntTarget extends XmlElement {
   AntTarget() { super.getName() = "target" }
 
   /** Gets the name of this Ant target. */
-  override string getName() { result = this.getAttributeValue("name") }
+  string getName() { result = this.getAttributeValue("name") }
```

Comment:
For future reference: It looks like this accidentally fixed a bug with this class. Previously the `override` repurposed the inherited `getName()`. And combined with the characteristics predicate of `AntTarget` being so unspecific, this actually broke the `XmlElement#getName()` of completely unrelated `<target>` occurrences in XML (even in Maven POMs) it seems.

So it is a good thing that `XmlElement` is now `final`, and that this here is not an `override` anymore.